decidim-core/app/cells/decidim/address/online.erb

```diff
@@ -2,8 +2,10 @@
   <%= icon "map-pin-line" %>
   <div class="address">
     <div class="address__location"><%= t(model.type_of_meeting, scope: "decidim.meetings.meetings.filters.type_values") %></div>
+      <% if meeting.online? && meeting.iframe_access_level_allowed_for_user?(current_user) %>
     <a href="<%= model.online_meeting_url %>" target="_blank" rel="noopener noreferrer" class="address__hints underline break-all">
       <%= model.online_meeting_url %>
+      <% end %>
```

Comment:
I think we should use the `model` variable instead of `meeting`. Also we should fix the indentation. I think this should be enough:

```suggestion
      <% if model.online? && model.iframe_access_level_allowed_for_user?(current_user) %>
        <a href="<%= model.online_meeting_url %>" target="_blank" rel="noopener noreferrer" class="address__hints underline break-all">
          <%= model.online_meeting_url %>
      <% end %>
```

Reply:
Could you test locally please to see if these changes reflect well with the rspec test I wrote and in the CI? 

I thought calling the Meeting model would be sufficient enough?

Reply:
> @andreslucena Could you test locally please to see if these changes reflect well with the rspec test I wrote and in the CI?
> 
> I thought calling the Meeting model would be sufficient enough?